pkgs/development/tools/build-managers/msbuild/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchurl, mono, dpkg }:
+
+stdenv.mkDerivation rec {
+  name = "msbuild-${version}";
+  version = "15.6";
+
+  src = fetchurl {
+    url = "https://download.mono-project.com/repo/ubuntu/pool/main/m/msbuild/msbuild_15.6+xamarinxplat.2018.01.17.14.14-0xamarin2+ubuntu1404b1_all.deb";
+    sha256 = "0jgw80p9dr9mi40i8vy22y3b5d4byr26s9gvs5xjjgdzqdrlj9fm";
+  };
+
+  buildInputs = [ dpkg ];
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  unpackPhase = "dpkg-deb -x $src .";
+  installPhase = "
+    substituteInPlace usr/bin/msbuild --replace /usr/bin/mono ${mono}/bin/mono
```

Comment:
FWIW you can do multiple replaces in one substituteInPlace.

Reply:
How?